### DISCUSSION TITLE
Add  reading-list-mover in Communique

### DISCUSSION THREAD

**Discussion by ghost** [Suzie97/Communique#24] [Q&A] [answered]
Hi all. 

Are you all thinking about supporting the [reading-list-mover](https://github.com/codebox/reading-list-mover) within the Communique in elementary-os? Is this possible with Communique? 

#### What is [reading-list-mover](https://github.com/codebox/reading-list-mover)?
"Copy bookmarks between Instapaper, Readability, Pocket, Pinboard, Delicious, Diigo, GitHub, StackOverflow and Twitter".

**Comment by Suzie97** [comment, marked as answer]
> Hi all. Are you all thinking about supporting the pocket.io within the Communique in elementary-os?

Hey there, I'm not sure what you mean by pocket.io. If you're talking about the read it lter service named pocket by Mozilla, Communique already supports it

**Reply by ghost** [reply]
> Hey there, I'm not sure what you mean by pocket.io. If you're talking about the read it lter service named pocket by Mozilla, Communique already supports it

Hi. I said getpocket... but... it was the: reading-list-mover. Please, read this again.
